src/unif.ml

```diff
@@ -314,6 +314,7 @@ and solve_aux : term -> term -> problems -> unif_constrs = fun t1 t2 p ->
           if !(s1.sym_rules) <> [] || List.same_length ts1 ts2
           then add_to_unsolved ()
           else error ()
+       | Priva -> failwith "implement me"
```

Comment:
There should be no Private sym_mode.

Reply:
Yes

Reply:
I assume the 'yes' concerns the remark above...

Reply:
Yes :-)